app/scripts/platforms/extension.js

```diff
@@ -2,18 +2,36 @@ import extension from 'extensionizer'
 import { createExplorerLink as explorerLink } from 'etherscan-link'
 import { getEnvironmentType, checkForError } from '../lib/util'
 import { ENVIRONMENT_TYPE_BACKGROUND } from '../lib/enums'
+import pify from 'pify'
+
+const CHROME_PIFY_OPTS = {
+  errorFirst: false,
+}
 
 class ExtensionPlatform {
 
+  constructor () {
+
+    if (extension.tabs) {
```

Comment:
I had to add this conditional to get the integration tests to pass.

If anyone can tell me how to mock this value, please let me know, and I'll do so.